modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java

```diff
@@ -131,4 +131,13 @@ public boolean isDefaultStorageType() {
 		return false;
 	}
 
+	@Override
+	public boolean unmodifiableSystemObject() {
```

Comment:
@mateussgn sorry, I think we misunderstood here. This method should still be called `isUnmodifiableSystemObject`

Reply:
reach me if you have any doubt

Reply:
Ooh, I'm sorry. I'll change the name.